test/browser/support/TestHarness.js

```diff
@@ -84,89 +84,63 @@ module.exports = class TestHarness {
       keyList = [];
       for (i = 0; i < keys.length; i++) {
         keyList.push(
-          listObjectsParsedResponse?.ListBucketResult?.Contents.at(i).Key
+          listObjectsParsedResponse?.ListBucketResult?.Contents.at(i).Key,
         );
       }
     } while (keys.length < txmaEventSize);
     return keyList;
   }
 
-  async validateTxMAEventData(keyList, journeyType) {
+  async getTxMAEventData(keyList) {
+    let obj = {};
     let i;
-    let valid = Boolean;
-    
     for (i = 0; i < keyList.length; i++) {
-      const getObjectResponse = await this.HARNESS_API_INSTANCE.get(
+      const txmaEventBody = await this.HARNESS_API_INSTANCE.get(
         "/object/" + keyList[i],
-        {}
+        {},
       );
-      console.log(JSON.stringify(getObjectResponse.data, null, 2));
-      const eventName = getObjectResponse.data.event_name;
-      const Ajv = require("ajv").default;
-      const AjvFormats = require("ajv-formats");
-      const ajv = new Ajv({ strictTuples: false });
+      console.log(JSON.stringify(txmaEventBody.data, null, 2));
```

Comment:
Do we need this log? It creates a fair amount of noise so just want to make sure it's helpful